src/frontend/src/utils/multiWebAuthnIdentity.ts

```diff
@@ -64,6 +67,14 @@ export class MultiWebAuthnIdentity extends SignIdentity {
       return this._actualIdentity.sign(blob);
     }
 
+    const currentUrl = new URL(window.location.origin);
+    const userAgent = window.navigator.userAgent;
+    const rorDomains = relatedDomains();
+    const rpId =
+      DOMAIN_COMPATIBILITY.isEnabled() && supportsWebauthRoR(userAgent)
+        ? findWebAuthnRpId(currentUrl.origin, this.credentialData, rorDomains)
```

Comment:
I suppose you can use `window.location.origin` here directly?

Reply:
Yes, but I wanted it to be readable. Do you think that will be more readable?

Reply:
No strong opinion on my side. Done!